ast2vec/tests/test_repo2coocc.py

```diff
@@ -45,10 +45,11 @@ def test_asdf(self):
             validate_asdf_file(self, file.name)
 
     def test_linguist(self):
+        # Make sure provide path to linguist is not pointing to an executable
         with self.assertRaises(FileNotFoundError):
             Repo2Coocc(linguist="xxx", timeout=600)
         with self.assertRaises(FileNotFoundError):
-            Repo2Coocc(linguist=__file__, timeout=600)
+            Repo2Coocc(linguist=os.path.dirname(__file__), timeout=600)
```

Comment:
The point of this test is to check for missing enry. However, repo2.base.py raises FileNotFoundError only when the provided path is not an executable (otherwise shutil.which will point to the executable).